images/tectonic-builder/Dockerfile

```diff
@@ -32,7 +32,7 @@ RUN mkdir -p ${HOME} && \
     chmod 777 -R ${HOME} && \
     apt-get update && \
     apt-get install --no-install-recommends -y -q \
-    build-essential sudo curl wget git autoconf automake unzip libtool jq awscli gnupg1 \
+    build-essential sudo curl wget git autoconf automake unzip libtool jq gnupg1 \
```

Comment:
This `awscli` is distinct from the `awscli` image you're dropping from `config.tf`.  I think we should drop the `config.tf` entry, since you're dropping its `rm-assets.sh` consumer.  *This* entry was added in 084ff71be (coreos/tectonic-installer#980), and [the motivation][1] was supporting role changes in smoke tests (now over [here][2]).  So I think we want to leave this line alone in this PR.

[1]: https://github.com/coreos/tectonic-installer/pull/980#issue-124037231
[2]: https://github.com/openshift/installer/blob/00066a96ddf772d68a31b7aba317ce7dee7160ca/tests/run.sh#L62-L64

Reply:
:+1: Sorry, I just get so excited when I can delete things.